packages/slice-machine/components/CustomTypeTable/ctPage.tsx

```diff
@@ -0,0 +1,64 @@
+import { StatusBadgeWithTooltip } from "@components/Badge/StatusBadgeWithTooltip";
```

Comment:
is this path easier to write?

Reply:
yes, changed to `../Badge/StatusBadgeWithTooltip` :)